Crucible.GameAPI/CrucibleIngredient.cs

```diff
@@ -430,9 +434,9 @@ public void SetStack(IEnumerable<CrucibleIngredientStackItem> rootItems)
             prefab.AddComponent<Rigidbody2D>();
 
             var stack = prefab.AddComponent<Stack>();
-            stack.inventoryItem = this.InventoryItem;
             var stackTraverse = Traverse.Create(stack);
-            stackTraverse.Property<ItemFromInventoryController>("SoundController").Value = new global::PotionCraft.ObjectBased.Stack.SoundController(stack, this.Ingredient.soundPreset);
+            stackTraverse.Property<InventoryItem>("InventoryItem").Value = this.InventoryItem;
+            stackTraverse.Property<ItemFromInventoryController>("SoundController").Value = new global::PotionCraft.ObjectBased.Stack.SoundController(stack, this.Ingredient.soundPreset); // TODO this is causing a null ref error in the constructor
```

Comment:
Is this TODO still relevant?  Do we still have a crash here?

Reply:
Removed it as that issue was fixed on a different line